### DISCUSSION TITLE
Is mosec available on windows?

### DISCUSSION THREAD

**Discussion by ynkim0** [mosecorg/mosec#439] [Q&A]
### Describe the bug

At first, Thanks for your great work. I can download and use mosec easily on ubuntu, but there are few thing got wrong on windows. Can you help me with this?

1. On windows, ForkContext is not working(ImportError: cannot import name 'ForkContext' from 'multiprocessing.context'). So I just removed ForkContext from runtime and only using SpawnContext. Isn't it okay?
2. I made virtual environment and used the pip tool on both windows and linux to download the 0.7.2 version of mosec. But on windows, there was no bin folder in virtual env. It only has "mixin", "plugins", "utils" folder. So FileNotFoundError occured in mosec/runtime 302 line (self._process = subprocess.Popen([self._server_path] + self._controller_args())). How should I solve it?

### To Reproduce

on windows 10, python 3.9

### The mosec version

0.7.2

**Comment by kemingy** [comment]
Thanks for asking.

`mosec` doesn't support Windows because:
- we use the Unix domain socket, which is not **fully** supported on Windows, check https://github.com/rust-lang/rust/issues/56533
- SpawnContext is mainly used in `mosec` so I guess it's okay, but I haven't tested it yet

Meanwhile, we don't have a Windows release due to the previous reasons. That's why the package you have downloaded doesn't contain the binary file. It's just a source code distribution.
